cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2744,6 +2730,19 @@
   </property>
 
   <property>
+    <name>security.authorization.propagate.privileges</name>
+    <value>true</value>
```

Comment:
Should we have the default as `false`, since no propagation is the more correct behavior? For backward compatibility, user can override this to `true`.